packages/office-ui-fabric-react/src/utilities/selection/Selection.ts

```diff
@@ -76,6 +80,26 @@ export class Selection implements ISelection {
     }
   }
 
+  public isModal(): boolean {
+    return this._isModal;
+  }
+
+  public setModal(isModal: boolean): void {
+    if (this._isModal !== isModal) {
+      this.setChangeEvents(true);
```

Comment:
Check the params here; this should be false and then followed by true.

Maybe we should throw when suppressioncount goes below 0.